### ISSUE TITLE
`templateFile` cannot parse image

### ISSUE THREAD

**Issue by aminimalanimal** [amwmedia/plop#86]
Is plop capable of duplicating an image to a different folder?

I'm trying to use a default image as a `templateFile` in my actions, but the command line chokes on the weird characters inside.

```
Lexical error on line 1. Unrecognized text.
����JFIFHH��C
----^
```

I assume this is occurring because plop is trying to walk through the file as though it is a handlebars template. I don't see any other option for straight-up duplication in the documentation. Is there a way to do this?

```
{
  type: 'add',
  path: '../{{folder}}/default.jpg',
  templateFile: './default.jpg'
},
```

**Comment by amwmedia** [comment]
the `add` action is used to generate new files from handlebar templates only. However, you can use [custom action functions](https://plopjs.com/documentation/#custom-action-function-) along side action objects. So, what you want to do could be achieved using an [npm package like `copy`](https://www.npmjs.com/package/copy)...

```javascript
const copy = require('copy');
plop.setGenerator('name', {
  ...
  actions: [
    { ... },
    function (answers) {
      const src = __dirname + '/default.jpg';
      const dest = plop.renderString(__dirname + '/{{folder}}/default.jpg', answers);
      copy.sync(src, dest);
      return 'copied default.jpg';
    }
  ]
});
```

Seems like the community would really benefit from a [3rd party action](https://plopjs.com/documentation/#setactiontype) for copying files 😉

**Comment by aminimalanimal** [comment]
I appreciate the help. I got it working with some minor changes to your snippet. For posterity's sake and for anyone that happens upon this when looking for a similar solution, I figured I'd note them here:
- the [version of `copy` I found](https://www.npmjs.com/package/copy) doesn't seem to have a `sync` method
- it seems to require you to pass an options argument with `{flatten: true}` in it so that it doesn't copy the directory folders into the output path as well

That put the final version at something like:
```
      function (answers) {
        const src = __dirname + '/lib/up_arrow.svg';
        const dest = plop.renderString(__dirname + '/{{folder}}/default.jpg', answers);
        copy(src, dest, {flatten: true});
        return 'copied default.jpg';
      },
```

I was strongly considering writing a 3rd party action, but don't have the time now. Maybe I'll come back to this problem again in the future and opt to go for it.
